bigbluebutton-html5/imports/startup/client/auth.js

```diff
@@ -46,7 +46,10 @@ export function authenticatedRouteHandler(nextState, replace, callback) {
   Auth.authenticate()
     .then(callback)
     .catch(reason => {
-      logClient('error', { error: reason, method: 'authenticatedRouteHandler', credentialsSnapshot });
+      logClient('error', {
+        error: reason,
+        method: 'authenticatedRouteHandler',
+        credentialsSnapshot, });
```

Comment:
I would close `});` on the next line, it looks more accurate. But it's up to you.